irohad/main/irohad.cpp

```diff
@@ -211,6 +212,19 @@ int main(int argc, char *argv[]) {
   std::signal(SIGTERM, handler);
   std::signal(SIGQUIT, handler);
 
+  // Check that torii and internal ports are not in use
+  if (checkTcpPortInUse(kListenIp, config[mbr::ToriiPort].GetUint())) {
```

Comment:
Port is usually has represented by the `short` type, check the `checkTcpPortInUse` signature as well